app/services/importers/new_court_case.rb

```diff
@@ -8,8 +8,7 @@ def initialize(link)
       @case_number = link.text
       @case_types = CaseType.oscn_id_mapping
       @counties = County.pluck(:name, :id).to_h
-      @uri = URI(link['href'])
-      @params = CGI.parse(uri.query)
+      @params = CGI.parse(URI(link['href']).query).map { |k,v| [k.downcase, v.first] }.to_h
```

Comment:
There was a bug where casemasterID could be in the URL as camelcase or all lowercase.